src/inference/src/dev/converter_utils.hpp

```diff
@@ -22,11 +24,16 @@ std::shared_ptr<const ov::Model> convert_model(const InferenceEngine::CNNNetwork
 std::shared_ptr<::InferenceEngine::IInferencePlugin> convert_plugin(const std::shared_ptr<::ov::IPlugin>& plugin);
 std::shared_ptr<::ov::IPlugin> convert_plugin(const std::shared_ptr<::InferenceEngine::IInferencePlugin>& plugin);
 
-std::shared_ptr<::InferenceEngine::IExecutableNetworkInternal> convert_compiled_model(
+OPENVINO_RUNTIME_API std::shared_ptr<::InferenceEngine::IExecutableNetworkInternal> convert_compiled_model(
     const std::shared_ptr<::ov::ICompiledModel>& model);
 std::shared_ptr<::ov::ICompiledModel> convert_compiled_model(
     const std::shared_ptr<::InferenceEngine::IExecutableNetworkInternal>& model);
 
+OPENVINO_RUNTIME_API std::shared_ptr<::InferenceEngine::IInferRequestInternal> convert_infer_request(
+    const std::shared_ptr<::ov::IAsyncInferRequest>& request);
+OPENVINO_RUNTIME_API std::shared_ptr<::ov::IAsyncInferRequest> convert_infer_request(
+    const std::shared_ptr<::InferenceEngine::IInferRequestInternal>& request);
```

Comment:
do we use these methods outside of inference library?

Reply:
Yes it is temporary change for template plugin.